### DISCUSSION TITLE
discuss release process

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#523] [Team Posts]
Current release process is done manually and requires us to interrupt you guys. We would be glad to integrate circleci and automated release, but for that we need an access to this repository settings. :( If you are tentative regarding this, we could discuss the better solution that you are comfortable with.

**Comment by rbotzer** [comment]
Agreed, and I should have done it a while back, sorry. You have Admin now, which should give you access to the settings. Tell me if you need assistance with anything.

**Comment by Aloren** [comment]
@rbotzer We have started the process of circle ci integration for spring-data-aerospike. Aerospike-community owners are requested for access from CircleCi, please confirm it once you receive the request. Thanks.

**Comment by rbotzer** [comment]
Just approved it. Let me know if anything else comes up.

**Comment by Aloren** [comment]
So fast! Thank you!
